### PR TITLE
Add heroku to publishing options

### PR DESCRIPTION
**Pull request by GabrielDierks**
Tested myself, a way to easily host static expo apps as a website on Heroku.

# Why

# How

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by GabrielDierks** [comment]
@EvanBacon would be nice i think to add this, since many are still using Heroku

File: docs/pages/distribution/publishing-websites.md [hunk 1]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
...
+  "clean_urls": false,
+  "routes": {
+    "/**": "index.html"
+  }
+}
+```
+
+### Connect your app with Heroku
+
+- Create a new Heroku app
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:284]: Link to heroku or wherever the user can create the app.

File: docs/pages/distribution/publishing-websites.md [hunk 2]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
     }
   ]
 ```
+
+## [Heroku hosting](https://dashboard.heroku.com/apps/)
+
+### static.json
+
+- Add a `static.json` to select the web-build folder:
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:261]:
```suggestion
- Add a `static.json` to select the `/web` folder:
```

File: docs/pages/distribution/publishing-websites.md [hunk 3]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
...
+
+```js
+{
+  "root": "web-build/",
+  "clean_urls": false
+}
+```
+
+- To enable client side routing add `"routes"`:
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:270]:
```suggestion
- To enable client side routing add the `"routes"` object to the `static.json`:
```

File: docs/pages/distribution/publishing-websites.md [hunk 4]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
...
+    "/**": "index.html"
+  }
+}
+```
+
+### Connect your app with Heroku
+
+- Create a new Heroku app
+
+- Connect your app via Heroku CLI or Github to deploy it
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:286]: Please add more info here.

File: docs/pages/distribution/publishing-websites.md [hunk 5]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
...
+
+
+### Heroku buildpack static
+
+- Add the `heroku-buildpack-static`. Go to Settings click on `Add buildpack` and paste [https://github.com/hone/heroku-buildpack-static](https://github.com/hone/heroku-buildpack-static) into the search bar and click `Save changes`.
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:291]: > Add the `heroku-buildpack-static`

No idea what this means

Comment (EvanBacon) [reply docs/pages/distribution/publishing-websites.md:291]:
```suggestion
- Add the `heroku-buildpack-static`. Go to Settings click on `Add buildpack` and paste [heroku-buildpack-static](https://github.com/heroku/heroku-buildpack-static) into the search bar and click `Save changes`.
```

Don't link a link and use the link as the preview.

File: docs/pages/distribution/publishing-websites.md [hunk 6]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
...
+
+> Remove all other buildpacks if there are.
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:293]: Not sure what this means either, please elaborate.

File: docs/pages/distribution/publishing-websites.md [hunk 7]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
...
+
+- Deploy your app
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:295]: How? Please add the command required to deploy the app.

File: docs/pages/distribution/publishing-websites.md [hunk 8]
```diff
@@ -253,3 +253,43 @@ In case you want to change the header for hosting add the following config in `h
     }
   ]
 ```
+
+## [Heroku hosting](https://dashboard.heroku.com/apps/)
```
Comment (EvanBacon) [inline docs/pages/distribution/publishing-websites.md:257]:
```suggestion
## [Heroku](https://www.heroku.com//)
```

**Pull request closed without merging**